src/Uno.UWP/UI/Color.cs

```diff
@@ -22,14 +24,7 @@ private Color(byte a, byte r, byte g, byte b)
 			B = b;
 		}
 
-		public override bool Equals(object o) {
-			if(o is Color color)
-			{
-				return Equals(color);
-			}
-
-			return false;
-		}
+		public override bool Equals(object o) => o is Color color && Equals(color);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename parameter 'o' to 'obj' to match the base class declaration.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5417387)